docs/source/developer_guide/tutorial.rst

```diff
@@ -187,20 +187,34 @@ Alternatively, using ``tox``::
 Adding a new Observable
 =======================
 
-Additional built-in observable quantities may be added for use by the ``janus_core.processing.correlator.Correlation`` class. These should conform to the ``__call__`` signature of ``janus_core.helpers.janus_types.Observable``. For a user this can be accomplished by writing a function, or class also implementing a commensurate ``__call__``.
+Additional built-in observable quantities may be added for use by the ``janus_core.processing.correlator.Correlation`` class. These should extend ``janus_core.processing.observables.Observable``. The abstract method ``__call__`` should be implemented to obtain the values of the observed quantity from an ``Atoms`` object:
 
-Built-in observables are collected within the ``janus_core.processing.observables`` module. For example the ``janus_core.processing.observables.Stress`` observable allows a user to quickly setup a given correlation of stress tensor components (with and without the ideal gas contribution). An observable for the ``xy`` component is obtained without the ideal gas contribution as:
+.. code-block:: python
+
+    @abstractmethod
+    def __call__(self, atoms: Atoms) -> list[float]
```

Comment:
I think walking through how you actually go about constructing this function might be useful, too. This could be based on one of the existing methods, or maybe a relatively trivial example

Reply:
I can expand the stress and velocity sections to detail more about how it was done there

Reply:
along with the other doc comment above, I have written the docs walking through writing the ```Stress``` and ```Velocity``` methods and their meaning

Reply:
Thanks!